.github/workflows/federation-v1.workflow.yaml

```diff
@@ -80,8 +80,8 @@ jobs:
       time: ${{ github.event_name == 'pull_request' && '30s' || '400s' }}
       baseDir: federation-v1
       runner: ${{ needs.decide-runner.outputs.runner }}
-      cpuLimit: '1'
-      memoryLimit: '1gb'
+      cpuLimit: '2'
```

Comment:
this one runs on 1 CPU on purpose. please revert. 
It's there to compare the difference between `constants-vus-subgraphs-delay` and `constants-vus-subgraphs-delay-resources`

Reply:
I am just experimenting things in this PR :)